source/guides/non-denizen/resource-packs.md

```diff
@@ -0,0 +1,248 @@
+
+For each file, you will need the data: `"name":"FILEPATH/FILENAME"`, excluding the file's extension.
+Optionally, you can manually adjust the following valid properties of the sound:
+1) volume - The volume the sound will be played as. 
+    - Default is `1.0`; Valid volume ranges from `0.0` to `1.0`; where `1.0` is the loudest it may be played at.
+    - The Volume value accepts higher values using Denizen's PlaySound, however not by increasing the volume. It increases the audible distance the sound may be heard from. Ie; volume 5 can be heard from five chunks away.
+2) pitch - The pitch the sound plays at, altered from it's original `.ogg` form.
+    - Default is `1.0`; Valid pitches range from `0.0` to `2.0`; where `0.0` is high-pitched and `1.0` will be low-pitched.
+3) weight - The chance that this sopund will be selected as opposed to randomly.
+    - Default is 0; Only accepts valid integers, not adjustable within Denizen.
+4) stream - Determines if the sound should be streamed from it's file.
+    - Default is false
+    - Recommended to set this value as `true` if the sound is longer than two seconds to avoid lag.
+    - Use this sparingly; it's not optimal to specify everything true.
+    - This is used with all music disks.
+5) preload - Determines if the sound should be loaded when loading the pack, as opposed to when the player plays the sound. 
+    - Default is false
+    - Used for ambient noises.
+6) attenuation_distance - Determines the reduction rate based on distance.
+    - Default is `1.0`
+    - Used by portals, beacons, conduits
+7) type - determines if a pre-defined event fires this sound.
+    - Default is `sound`
+    - used for things like being under-water, in a cave, near a beacon, near a beehive.
+
+### Inside The Blockstates Folder Directory
+Directory: `.minecraft\resourcepacks\MyResourcePack\assets\minecraft\blockstates\`
+To modify each individual block-state of an item, you must specify each individual blockstate.
+Additional blockstates cannot be specified. When specifying blockstate models, the relative folder directs to the `Models` directory, located at `\assets\minecraft\models\`
+Adjusting these are not traditional, therefor not covered further in this guide.
+
+### Inside The Models Folder Directory
+Directory: `.minecraft\resourcepacks\MyResourcePack\assets\minecraft\models\`
+To "Create" new items, you will need to modify existing items within Minecraft. This can and was previously done with Durability, but optimally utilized with `custom_model_data` that was implemented in Minecraft 1.14. There are three object model types for model data: ,`Block States`,`Block Models` and `Item Models`. Block States and Block Model data are not covered in this guide.
```

Comment:
You forgot the oxford comma, but chose to include the rare pre-comma.
Also missing a space-after-comma.